packages/tradershub/src/components/TradingAccountsList/TradingAccountsList.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useActiveTradingAccount, useAuthorize, useTradingAccountsList } from '@deriv/api';
+import { qtMerge, Text } from '@deriv/quill-design';
+import { IconToCurrencyMapper } from '../../constants/constants';
+
+const TradingAccountsList = () => {
+    const { data: tradingAccountsList } = useTradingAccountsList();
+    const { data: activeAccount } = useActiveTradingAccount();
+    const { switchAccount } = useAuthorize();
+
+    return (
+        <div className='flex w-[500px] h-[350px] flex-col items-center rounded-400 bg-system-light-primary-background'>
+            <div className='flex flex-col items-start self-stretch flex-1 p-400 gap-200'>
+                {tradingAccountsList
+                    ?.filter(account => !account.is_virtual)
+                    .map(account => {
+                        const iconCurrency = account.currency ?? 'USD';
+                        return (
+                            <button
```

Comment:
Had to use button here instead of div cause sonarcloud was complaining due to not able to identify what type of role that div is

Reply:
how about giving it a role attribute?

Reply:
Sonar cloud will still complain

Reply:
yes 😭